components/DetailsCard/index.js

```diff
@@ -138,19 +143,18 @@ export default function DetailsCard() {
         {activities.description !== "" && (
           <StyledDescription>{activities.description}</StyledDescription>
         )}
+        <StyledJoinButton isJoined={joinState.isJoined} onClick={handleJoin}>
+          {joinState.isJoined ? "Disjoin" : "XX Join"}
+        </StyledJoinButton>
         <StyledButtonBox>
-          <StyledEditButton
-            onClick={() => {
-              setIsEditMode(!isEditMode);
-            }}
-          >
+          <StyledButton onClick={() => (window.location.href = "/")}>
```

Comment:
this should not be a button, but a Link, since you navigate to another page.

It would be a button if some logic would have to be applied before navigating.